community/contributing/docs/local-clone.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: single
+title: Make a Change Using a Local Clone
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+If your changes are larger than a few lines or involve multiple files, you should fork the `spinnaker/spinnaker.github.io` repository and work from a local clone. Make sure you have [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed on your computer and have configured your GitHub account. See the [GitHub Help](https://help.github.com/en/github) for details.
+
+## Fork the Spinnaker documentation repository
+
+1. Navigate to the `spinnaker/spinnaker.github.io` [repository](https://github.com/spinnaker/spinnaker.github.io/) with a web browser.
+1. Click **Fork**.
+
+## Create a local repository and set the upstream repository
+
+1. In a terminal window, clone your fork:
+
+   ```bash
+   git clone git@github.com/<github-username>/spinnaker.github.io.git
+   ```
+
+1. Navigate to the new `spinnaker.github.io` directory. Set the `spinnaker/spinnaker.github.io` repository as the `upstream` remote:
+
+   ```bash
+   cd spinnaker.github.io
+
+   git remote add upstream https://github.com/spinnaker/spinnaker.github.io.git
+   ```
+
+1. Confirm your `origin` and `upstream` repositories:
+
+   ```bash
+   git remote -v
+   ```
+
+   Output is similar to:
+
+   ```bash
+   origin	git@github.com:<github-username>/spinnaker.github.io.git (fetch)
+   origin	git@github.com:<github-username>/spinnaker.github.io.git (push)
+   upstream	https://github.com/spinnaker/spinnaker.github.io.git (fetch)
+   upstream	https://github.com/spinnaker/spinnaker.github.io.git (push)
+   ```
+
+## Update your local repository
+
+Make sure your local repository is current before you start making changes. Fetch commits from your fork's `origin/master` and `spinnaker/spinnaker.github.io`'s `upstream/master`:
+
+   ```bash
+   git fetch origin
+   git fetch upstream
+   ```
+
+   This workflow is different from a source code commit workflow . You do not need to rebase your local copy of `master` with `upstream/master` before pushing updates to your fork. In this documentation workflow, you create a working branch that tracks changes to `upstream/master` rather than `origin/master`.
```

Comment:
Source code committers  usually rebase their local master against upstream and push the changes to their fork origin before creating each working branch... or at least that's how I contributed source code to projects using GitHub. 
I put this paragraph in here so engineers know contributing to docs has a different workflow, which is simpler than what engineers use.

Reply:
I suspect anyone who has an existing GitHub workflow they're already familiar with is never going to read this page anyway. (Personally, I would just ignore this advice and use the same workflow I use everywhere else. It will work just as well and trying to remember to use a different workflow for this one repository just seems more confusing to me, not less.) So is it just adding confusion for this page's primary audience (people who are fairly new to git/GitHub)?

I dunno, just a thought.

Reply:
I agree w/ @plumpy. If someone is already used to committing to GitHub, they won't be using this. Telling them this doesn't add anything.

Reply:
This is the docs workflow that k8s docs uses. Coming from development,  I asked WHY in docs I didn't have to rebase my fork before branching, and why use `git checkout -b <your-working-branch> upstream/master` instead of just `git checkout -b <your-working-branch>`.  The docs team there had no idea, so I had to track down somebody who did. Inquiring minds   want to know!

I added a sentence to the working branch section and removed from "Update your local repository".

Reply:
I checked with a prominent (former) K8s contributor, and with Maggie, and I'm satisfied that this workflow is fine. Carry on.